code/Model/Recipient/EmailRecipient.php

```diff
@@ -562,7 +568,7 @@ public function getEmailTemplateDropdownValues()
         $templates = [];
 
         $finder = new FileFinder();
-        $finder->setOption('name_regex', '/^.*\.ss$/');
+        $finder->setOption('name_regex', static::config()->get('email_template_regex'));
```

Comment:
Made configurable so if someone wants to swap out to a different rendering engine they can update this to `/^.*\.twig$/` or whatever.

Really this whole thing should just use the regular `template/` dir and let the regular template lookup process occur - but changing that is out of scope.